src/app/static/src/app/root.ts

```diff
@@ -107,6 +115,10 @@ const resetState = (store: Store<RootState>): void => {
             if (type[0] == "modelRun" && ModelRunUpdates.includes(type[1] as ModelRunMutation)) {
                 store.commit(RootMutation.ResetOutputs);
             }
+
+            if (type[0] == "modelCalibrate" && ModelCalibrateUpdates.includes(type[1] as ModelCalibrateMutation)) {
+                store.commit(RootMutation.ResetDownload);
```

Comment:
This line still seems to be untested: your should be able add a test [here](https://github.com/mrc-ide/hint/blob/master/src/app/static/src/tests/components/app.test.ts)

Reply:
Actually this was tested in roots/mutations but I have added another test to app.tests

Reply:
This is a plugin though, rather than a mutation, so it wouldn't be covered by the mutation test - codecov didn't seem to think so anyway! But fixed now by your new test :+1: